targets/mbedos5/mbed_app.json

```diff
@@ -0,0 +1,7 @@
+{
+    "target_overrides": {
+        "NRF52_DK": {
+            "target.uart_hwfc": 0
+        }
+    }
+}
```

Comment:
Only reason I included it was to make sure that standard I/O on the NRF52 doesn't cause the program to hang if you aren't connected to the serial port (seems like hanging is the default behaviour in mbed OS for some reason)